tests/nightwatch/login.js

```diff
@@ -0,0 +1,17 @@
+describe('Login into nextcloud end-to-end test', function() {
+	before(browser => browser.navigateTo(browser.launchUrl))
+
+	it('arrives on the home page and submits credentials with login form', function(browser) {
+		browser
+			.waitForElementVisible('body')
+			.assert.visible('input#user')
+			.setValue('input#user', browser.globals.adminLogin)
+			.setValue('input#password', browser.globals.adminPwd)
+			.click('button[type=submit]')
+	})
+	it('verifies that workspace icon is present', function(browser) {
```

Comment:
Can you replace `'verifies that workspace icon is present'` by `check that workspace icon is present` please ?